docs/standards/rpc-api.md

```diff
@@ -10,14 +10,109 @@ sidebar_position: 10
 This page is under active development.
 
 :::
+The [LUKSO Extension](../guides/browser-extension/install-browser-extension.md) uses the RPC API methods from the table below. The methods are grouped by category: signing methods, standard methods, and LUKSO-specific methods.
 
-The [LUKSO Extension](../guides/browser-extension/install-browser-extension.md) uses new RPC API which are described here.
+## Supported Methods
 
-## Methods
+<table>
+  <tr>
+    <td>Name</td>
+    <td>Type</td>
+  </tr>
+  <tr>
+    <td><a href="#eth_sign">eth_sign</a></td>
+    <td>signing</td>
+  </tr>
+  <tr>
+    <td><a href="#personal_sign">personal_sign</a></td>
+    <td>signing</td>
+  </tr>
+  <tr>
+    <td><a href="#eth_getAccounts">eth_getAccounts</a></td>
+    <td>standard</td>
+  </tr>
+  <tr>
+    <td><a href="#eth_requestAccounts">eth_requestAccounts</a></td>
+    <td>standard</td>
+  </tr>
+  <tr>
+    <td><a href="#eth_sendTransaction">eth_sendTransaction</a></td>
+    <td>standard</td>
+  </tr>
+  <tr>
+    <td><a href="#wallet_switchEthereumChain">wallet_switchEthereumChain</a></td>
+    <td>standard</td>
+  </tr>
+   <tr>
+    <td><a href="#up_addTransactionRelayer">up_addTransactionRelayer</a></td>
+    <td>LUKSO specific</td>
+  </tr>
+   <tr>
+    <td><a href="#up_import">up_import</a></td>
+    <td>LUKSO specific</td>
+  </tr>
+   <tr>
+    <td><a href="#up_generateLsp23Address">up_generateLsp23Address</a></td>
+    <td>LUKSO specific</td>
+  </tr>
+</table>
 
-### up_addTransactionRelayer
+## Signing
 
-Add a custom relayer.
+### eth_sign
+
+**_NOTE:_** We encourage developers to use `eth_sign` for signing purposes.
+
+While a security issue potentially existed in the initial implementation on Ethereum, the current implementation has no such potential exploit. As such usage of this method is preferable to [personal_sign](#personal_sign).
+
+This method returns a [EIP-191](https://eips.ethereum.org/EIPS/eip-191) signature over the data provided to the call.
+It requests that the user provides an Ethereum address that should sign the transaction as well as the data (encoded bytes) that are to be executed.
+
+Returns
+`string` - on a successful call the method returns a signature, a string representing hex encoded bytes or an error with code 4001 - if the user rejects the requets
+
+### personal_sign
+
+The personal_sign endpoint is enabled to allow for backward compatibility. However, its use is not recommended.
```

Comment:
Similar to how Hugo wrapped eth_sign into `` let's wrap all other pieces of code or RPC names into `` as well, like `personal_sign` here.

Those that are wrapped in HTML tags or are links `[text](link)` we can leave as is.